server.js

```diff
@@ -1,88 +1,118 @@
 'use strict';
 
-const _ = require('lodash');
-require('app-module-path').addPath('./shared');
-
-const jwt = require('jwt-simple');
-const cookieParser = require('cookie-parser');
-const bodyParser = require('body-parser');
-const express = require("express");
-const authRoutes = require('./server/auth_routes');
-//const userRoutes = require('./server/profile_routes');
-const passportSetup = require('./server/passport_setup');

```

Comment:
Actually, big problem here. Figured out why Google Login crashes: We are no longer importing `passport_setup.js` anywhere so it never gets run. Just importing it at all runs all the code inside.